pkg/backup/backup.go

```diff
@@ -1102,10 +1150,24 @@ func buildFinalTarball(tr *tar.Reader, tw *tar.Writer, updateFiles map[string]Fi
 	return nil
 }
 
-type tarWriter interface {
-	io.Closer
-	Write([]byte) (int, error)
-	WriteHeader(*tar.Header) error
+//type tarWriter interface {
```

Comment:
Did we want to remove this interface or implement/use it ?
If using it then I think we might need to rename the struct implementing the interface and implement the interface methods. 

Also from the commented code -- Did you want to do something like (to be more explicit):
```
func (tw *tarWriter) Lock() {
	tw.mu.Lock()
}

func (tw *tarWriter) Unlock() {
	tw.mu.Unlock()
}
```